docs/Manual/07 Plans - Vectorization and Parallelization.md

```diff
@@ -20,7 +20,7 @@ def _():
 
 plan = nest.create_plan(my_target)
 ```
-If we build `plan` as is, the resulting implementation would be equivalent to the following Python code:
+If we build the `plan` as is, the resulting implementation would be equivalent to the following Python code:
```

Comment:
`plan` is actually the name of the object above returned by the python snippet, so "build `plan`" seems more grammatical in this context.